packages/amplify-e2e-core/src/init/amplifyPush.ts

```diff
@@ -44,10 +44,12 @@ export const amplifyPush = async (cwd: string, testingWithLatestCodebase = false
   const pushArgs = ['push', ...(opts?.minify ? ['--minify'] : [])];
   // Test amplify push
   await spawn(getCLIPath(testingWithLatestCodebase), pushArgs, { cwd, stripColors: true, noOutputTimeout: pushTimeoutMS })
-    .wait('Are you sure you want to continue?')
+    .wait('Are you sure you want to continue "amplify push"?')
```

Comment:
is this correct change? 
If e2e tests pass then it's fine I guess.
Otherwise the `wait` argument can be subset of prompt for example `Are you sure you want to continue`.

Reply:
Oops, I accidentally commit this change..

Reply:
Since it passed the e2e test and it's sort of useful, I'll just leave it here

Reply:
all e2e tests?

Reply:
Not all e2e tests tho 🫢 

Ok...

Reply:
I guess it does matter https://github.com/aws-amplify/amplify-cli/blob/dev/packages/amplify-cli/src/extensions/amplify-helpers/push-resources.ts#L105

Reply:
yeah.
if you want to capture both variants then either
`.wait('Are you sure you want to continue')`
OR `.wait(/<some awesome regex to capture both/)` 
see https://github.com/aws-amplify/amplify-cli/blob/7bc0b5654a585104a537c1a3f9615bd672435b58/packages/amplify-e2e-core/src/utils/nexpect.ts#L75

Reply:
(not sure if we have 2 variants)

Reply:
To make it simple, I just removed the change 😂  4ad7631a005306761a8081fd052faf97d2ff4b15